build.sc

```diff
@@ -49,6 +49,7 @@ object scalanet extends ScalaModule with PublishModule {
     ivy"org.scodec::scodec-bits:1.1.12",
     ivy"org.scodec::scodec-core:1.11.4",
     ivy"org.scala-lang.modules::scala-parser-combinators:1.1.2"
+	ivy"com.madgag.spongycastle:core:1.56.0.0"
```

Comment:
spongycastle is actually special fork of bouncycastle designed for android and is no longer supported, it would be better to use bouncy castle version 1.64